android/app/src/main/AndroidManifest.xml

```diff
@@ -27,6 +30,9 @@
         </intent-filter>
       </activity>
       <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
+      <meta-data
+    android:name="com.google.android.nearby.messages.API_KEY"
+    android:value="AIzaSyCwOU9k7lloNkCQ5kdjzCSoyzyYWqtd-9I" />
```

Comment:
Why is this suddenly committed here? I haven't seen this put in this file before and I thought we would've seen it before.

Reply:
This was the commit to fix android nearby.